cedar-wasm/src/policies_and_templates.rs

```diff
@@ -0,0 +1,250 @@
+use std::str::FromStr;
+
+use cedar_policy::{Policy, PolicySet};
+use serde::{Deserialize, Serialize};
+
+use tsify::Tsify;
+use wasm_bindgen::prelude::*;
+
+#[derive(Tsify, Debug, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+#[tsify(into_wasm_abi, from_wasm_abi)]
+pub enum JsonToPolicyResult {
+    Success { policy_text: String },
+    Error { errors: Vec<String> },
```

Comment:
Probably want a more rich error type here, we started with strings and our in the process of moving to more and more structured.

Reply:
is there a specific type you recommend?

Reply:
On second thought, i think we might wanna keep this a string since there are two possible sources of errors here. Deserializing into serde_json::value can yield one type of Err, and then parsing the policy can yield another type. WDYT?

Reply:
`String`s are probably ok here for ease of (de)serializability. You could also use a simplified version of our [`ParseError` enum](https://github.com/cedar-policy/cedar/blob/687d0f5d62b80556dfaa2b2c296703c50c332b4f/cedar-policy-core/src/parser/err.rs#L48). (Note: that enum is not (de)serializable as-is because the error variants contain a lot of extra information.)